SPEC.md

```diff
@@ -71,7 +72,7 @@ It will then look for this executable in a list of predefined directories. Once
 - `CNI_ARGS`: Extra arguments passed in by the user at invocation time. Alphanumeric key-value pairs separated by semicolons; for example, "FOO=BAR;ABC=123"
 - `CNI_PATH`: Colon-separated list of paths to search for CNI plugin executables
 
-Network configuration in JSON format is streamed through stdin.
+Network configuration in JSON format is streamed through stdin. This means it is not tied to a particular file on disk and can contain information which changes between invocations.
```

Comment:
"streamed to the plugin through stdin" ?  Just to make it a bit clearer what we are talking about.